xml/System.Net/NetworkCredential.xml

```diff
@@ -79,6 +79,11 @@
 ## Remarks
  The <xref:System.Net.NetworkCredential> class is a base class that supplies credentials in password-based authentication schemes such as basic, digest, NTLM, and Kerberos. Classes that implement the <xref:System.Net.ICredentials> interface, such as the <xref:System.Net.CredentialCache> class, return <xref:System.Net.NetworkCredential> objects.
 
+> [!NOTE]
+> On Unix platforms, use of NTLM authentication may require installing the NTLM plugin for the GSSAPI layer used by the implemention. On Ubuntu distributions, the package is called `gss-ntlmssp`. Alternatively, applications may opt in for a managed implementation of NTLM by setting the `System.Net.Security.UseManagedNtlm` AppContext switch.
```

Comment:
Should we be documenting the AppContext switch, or the `_UseManagedNtlm` .csproj property? Switching the AppContext at runtime may lead to undefined behavior, and it won't necessarily work for trimmed scenarios.

Reply:
updated